omero_downloads.html

```diff
@@ -93,7 +93,7 @@
                             apply for an <a
                                 href="http://qa.openmicroscopy.org.uk/registry/demo_account/"
                                 >account on our Demo server</a> or download the
-                                <a href="#va">Virtual Appliance</a> to install
+                                <a href="http://www.downloads.openmicroscopy.org/latest/omero-virtual-appliance">Virtual Appliance</a> to install
```

Comment:
Maybe remove `http://www.downloads.openmicroscopy.org` since this page is on the same server? Then if someone goes to https they won't get switched to http.